package.json

```diff
@@ -47,18 +47,19 @@
   "devDependencies": {
     "@size-limit/preset-small-lib": "^7.0.8",
     "@tsconfig/recommended": "^1.0.1",
-    "axios": "^0.26.0",
+    "axios": "^0.26.1",
```

Comment:
would you consider also dropping axios in favor of so like unfetch?
I personally prefer working with axios as well, but it's quite heavy on the bundle when not needed for anything else.

https://bundlephobia.com/package/axios@0.26.1
https://bundlephobia.com/package/isomorphic-unfetch@3.1.0

Reply:
Those are dev dependancies only. I let you discover the new SDK through the readme: https://github.com/paraswap/paraswap-sdk/blob/sdk-refactoring/README.md. Contributions are welcome for an `unfetch` wrapper.

Reply:
I would expect `unfetch` to just work with `fetch` wrapper, even if types may not quite match